README.md

```diff
@@ -230,11 +231,14 @@ run HarbourBridge on the modified pg_dump/mysqldump output.
 
 HarbourBridge generates several files as it runs:
 
-- Schema file (ending in `schema.txt`): contains the generated Spanner
-  schema, interspersed with comments that cross-reference to the relevant
+- Schema DDL file (ending in `schema.ddl.txt`): contains the generated Spanner
+  schema DDL, interspersed with comments that cross-reference to the relevant
   PostgreSQL/MySQL schema definitions.
 
-- Report file (ending in `report.txt`) contains a detailed analysis of the
+- Schema conversion file (ending in `schema.conv.json`): contains all schema 
+conversion state endcoded as JSON. This file is written for `-schema-only` mode.
```

Comment:
I think we should write this file all of the time.
It captures the state/session of the tool.
I don't see a strong reason to just do it for schema-mode, and the fewer special-cases the better.

Moreover, I could see it being useful in turn-key mode:
a) run in turn-key mode and get a session.json
b) add some more data using data-only mode, and use the session.json from step (a) to get the schema and table/col mappings.

Reply:
Got it ! I will make the change to generate this file all the time.